keylime/config.py

```diff
@@ -180,31 +183,27 @@ def get_config(component: str) -> RawConfigParser:
             if os.path.exists(CONFIG_ENV[component]):
                 if os.path.isfile(CONFIG_ENV[component]):
                     config_files = _config[component].read(CONFIG_ENV[component])
-                    print(f"Reading configuration from {config_files}")
+                    log_msg = f"Reading configuration from {config_files}"
+                    base_logger.debug(log_msg)
```

Comment:
nit: there is a difference between `base_logger.debug(f"Reading configuration from {config_files}")` and `base_logger.debug(f"Reading configuration from %s", config_files)`

In the first case the interpolation is always done, whatever the logger decides to output the string or not, but in the second the interpolation is done only if the string is emitted.

For this case the cost is almost the same, but as a general rule, for logs, we should always use the second form.

Reply:
Changed all of it (except for the one in line 246, where pylint complained and wanted to keep a bit of post-interpolation beautification)